packages/examples/sdk-frontend/automate.sh

```diff
@@ -0,0 +1,30 @@
+#! /bin/bash
+
+who=$(whoami)
+echo "Hey, $who!"
+
+sleep 1
+
+delnode=$(rm -rf node_modules/)
+sleep 1
+echo "Node Modules Deleted."
+
+sleep 1
+
+delnext=$(rm -rf .next/)
+sleep 1
+echo ".Next Deleted"
+
+sleep 1
+
+clean=$(yarn cache clean)
+sleep 1
+echo "Yarn Cache Cleaned."
+
+sleep 1
+install=$(yarn)
+sleep 1
+echo $install
+
```

Comment:
@samarendra-push Can we add 'yarn run dev' as well here? :)

Reply:
Sure!